core/src/main/scala/pureconfig/package.scala

```diff
@@ -143,9 +166,7 @@ package object pureconfig {
    */
   @throws[ConfigReaderException[_]]
   def loadConfigOrThrow[Config](path: Path)(implicit conv: ConfigConvert[Config], ct: ClassTag[Config]): Config = {
```

Comment:
The `ClassTag` is propagated all the way down to the `ConfigReaderException`, which uses it for to know the runtime name of the class for the error message.